kphp_polyfills.php

```diff
@@ -769,6 +769,51 @@ function memory_get_static_usage(): int {
 
 #endregion
 
+#region job workers
+
+interface KphpJobWorkerRequest {}
+interface KphpJobWorkerResponse {}
+
+class DummyKphpJobWorkerRequest implements KphpJobWorkerRequest {}
+
+/**
+ * @param KphpJobWorkerRequest $request
+ * @return int
+ */
+function kphp_job_worker_start(KphpJobWorkerRequest $request): int {
+  return 0;
```

Comment:
Let's add for all functions (except `is_kphp_job_workers_enabled()`) a critical error `critical_error("not implemented");`

Reply:
Added.